plugins/ci/pkg/ci/ci.go

```diff
@@ -416,39 +416,51 @@ func getGitInfo(repoName, baseBranch string) (gitInfo, error) {
 		return info, fmt.Errorf("%v: %v", "Please be sure to run the insights-ci script inside of a valid git repository, with the branch you are scanning checked out", err)
 	}
```

Comment:
we can probably remove this check, since now the user could just specify everything w/ environment variables